lightning_examples/finetuning-scheduler/finetuning-scheduler.py

```diff
@@ -348,6 +338,8 @@ def __init__(
                 "default".
         """
         super().__init__()
+        self.training_step_outputs = []
+        self.validation_step_outputs = []
```

Comment:
Do we actually consume these lists anywhere? I think we only append and then clear at the end of epoch. I would remove if they are redundant.

Reply:
Thanks for noticing. These were/are used in other more advanced versions of this example I run in other contexts (e.g. FSDP multi-phase scheduled fine-tuning) but I've now removed them from the basic example since they're not used as you observe.

Reply:
Very nice! I didn't know, if you want to keep them that's also fine with me.